packages/types/src/Bytes.ts

```diff
@@ -46,10 +42,10 @@ export default class Bytes extends U8a {
 
   toU8a (isBare?: boolean): Uint8Array {
     return isBare
-      ? this.raw
+      ? super.toU8a(isBare)
```

Comment:
Shouldn't this one also be `this` like below? (Same thing, bug may help in consistency?)

Reply:
No, I mean the following -

```
toU8a (isBare?: boolean): Uint8Array {
  return isBare
    ? this
    : u8aConcat(
        Compact.encodeU8a(this.length, DEFAULT_LENGTH_BITS),	       
        this
      );
}
```